perseo-main/src/main/java/com/telefonica/iot/perseo/Constants.java

```diff
@@ -24,7 +24,7 @@
  * @author brox
  */
 public final class Constants {
-    private Constants() { };
+    private Constants() {}
```

Comment:
Why this change is needed?

Reply:
Fixed in 05982ba4eefed043b201d8b95eb1c9b6190d2673

Reply:
I have had a look to 05982ba and it seems perseo-main/src/main/java/com/telefonica/iot/perseo/Constants.java file is not modified in that commit...

The fix seems to be pending.

Reply:
Still pending...